### ISSUE TITLE
Is it possible to get private album photos with PhotosetsInterface.getPhotos?

### ISSUE THREAD

**Issue by javagrl** [boncey/Flickr4Java#251]
### Subject of the issue
I get FlickrException: 1: Photoset not found

### Your environment
* 2.1.7
* 1.8

### Steps to reproduce
public void writeImagesToFile() {
		String setId = "72157688426576935";
		String secret =  FlickrDownloadProps.getProperties().getString("flickr.secret");
		String apiKey = FlickrDownloadProps.getProperties().getString("flickr.api.key");
		Transport transport = new REST();
		RequestContext requestContext = RequestContext.getRequestContext();
		flickr = new Flickr(apiKey, secret, transport);
		
		try {
AuthInterface ai = flickr.getAuthInterface();
			String filepath = "resources/";
			File authDir = new File(filepath);
			if (authDir.exists()) {
				FileAuthStore store = new FileAuthStore(authDir);
				store.retrieve("39955793@N07");
				System.out.println("got the auth store file");
				
			}
// seturl not used but you can plug it into browser. You will get a Woops! from Flickr because it is private
			String seturl="https://www.flickr.com/photos/39955793@N07/sets/72157688426576935";
			
PhotosetsInterface pi = flickr.getPhotosetsInterface();
			Photoset pset = pi.getInfo(setId);
			int count = pset.getPhotoCount();
			System.out.println("count: " + count);
			PhotoList<Photo> photoList  = pi.getPhotos(setId,30,1);
			for (Iterator<Photo> iter = photoList.iterator(); iter.hasNext();) {
				Photo photo = (Photo) iter.next();
				String id = photo.getId();
				String url = photo.getOriginalUrl();
				BufferedImage img = ImageIO.read(new URL(url));
			File file = new File("C://Users/acrace/" + photo.getTitle() + ".jpg");
			ImageIO.write(img, "jpg", file);
			}
			
		} catch (FlickrException | IOException e ) {
			// TODO Auto-generated catch block
			e.printStackTrace();
	
		} finally {
			//do stuff
		}
	}
### Expected behaviour
It's actually doing what I now know to expect it to do. I'm wondering if anyone can help me make a signed call to get all the photos in the album. Even if I have their Ids, I can call each one individually. Not exactly the most efficient way to do it but I have successfully downloaded one non-private image.

### Actual behaviour
com.flickr4java.flickr.FlickrException: 1: Photoset not found

### Log
```
Please copy paste the log here if available

```

**Comment by boncey** [comment]
Hello.

I think what's happening here is that the `id` taken from the URL is not the `id` for the method call.
It's a strange anomaly in the API - you need to use different ids.

You need to call `flickr.photosets.getList` first and use the ids from those results instead.
Hope this helps.

**Comment by javagrl** [comment]
Thanks, boncey. I still need help with signing the call. I was able to make the call to flickr.getPhotosetsInterface.getList(userId).getCount(); I get 60 when I do it as you described above on the Flickr site, I get 62. The photosetId is correct. That's how I got it in the first place. I need to get at the 2 private albums. If I don't sign on the Flickr site, I get User not found although I can get the non-private ones with flickr4java.
